### PR TITLE
[IMP] helpdesk_fieldservice: Add 'Service Orders' menu item (PM #576)

### PR DESCRIPTION
**Pull request by osi-scampbell**
This PR adds Service Orders to Helpdesk menu under the Operations parent menuitem. Also moves All Tickets menuitem under Operations parent as well.
Heirarchy:
Overview
Operations
*All Tickets
*Service Orders
Master Data
Reporting
Configuration

### CODE REVIEW

File: helpdesk_fieldservice/views/helpdesk_ticket_views.xml [hunk 1]
```diff
@@ -45,4 +45,22 @@
...
+              action="fieldservice.action_fsm_dash_order"
+              parent="menu_helpdesk_op"
+              sequence="20"/>
+
+    <data noupdate="1"><delete model="ir.ui.menu" id="helpdesk.helpdesk_ticket_menu_main"/></data>
+
+    <menuitem id="helpdesk_ticket_menu_main" 
+        action="helpdesk.helpdesk_ticket_action_main_tree"
+        sequence="10"
+        parent="menu_helpdesk_op"/>
```
Comment (max3903) [inline helpdesk_fieldservice/views/helpdesk_ticket_views.xml:62]: @osi-scampbell You don't need to delete the menu item to re-create it. Just change the parent with:

```
<menuitem id="helpdesk.helpdesk_ticket_menu_main" 
￼	  parent="menu_helpdesk_op"/>
```

File: helpdesk_fieldservice/views/helpdesk_ticket_views.xml [hunk 2]
```diff
@@ -45,4 +45,20 @@
...
+              sequence="10"/>
+
+    <menuitem id="menu_helpdesk_product"
+              name="Service Orders"
+              action="fieldservice.action_fsm_dash_order"
+              parent="menu_helpdesk_op"
+              sequence="20"/>
+
+    <menuitem id="helpdesk.helpdesk_ticket_menu_main" 
+        name="All Tickets"
```
Comment (max3903) [inline helpdesk_fieldservice/views/helpdesk_ticket_views.xml:61]:
```suggestion
```

Comment (osi-scampbell) [reply helpdesk_fieldservice/views/helpdesk_ticket_views.xml:61]: If we don't have that the menu shows as 'helpdesk.helpdesk_ticket_menu_main' where the name should be

Comment (osi-scampbell) [reply helpdesk_fieldservice/views/helpdesk_ticket_views.xml:61]: Helpdesk

File: helpdesk_fieldservice/views/helpdesk_ticket_views.xml [hunk 3]
```diff
@@ -45,4 +45,20 @@
             </field>
         </field>
     </record>
+
+    <menuitem id="menu_helpdesk_op"
+              name="Operations"
+              parent="helpdesk.menu_helpdesk_root"
+              sequence="10"/>
+
+    <menuitem id="menu_helpdesk_product"
```
Comment (max3903) [inline helpdesk_fieldservice/views/helpdesk_ticket_views.xml]:
```suggestion
    <menuitem id="menu_helpdesk_fsm_order"
```

**Review by max3903 — APPROVED** [review]

**Pull request merged**